cmd/recommend.go

```diff
@@ -49,4 +49,5 @@ func init() {
 	recommendCmd.Flags().StringVarP(&recommendOptions.OutDir, "outdir", "o", "out", "output folder to write policies")
 	recommendCmd.Flags().StringVarP(&recommendOptions.ReportFile, "report", "r", "report.txt", "report file")
 	recommendCmd.Flags().StringSliceVarP(&recommendOptions.Tags, "tag", "t", []string{}, "tags (comma-separated) to apply. Eg. PCI-DSS, MITRE")
+	recommendCmd.Flags().StringVarP(&recommendOptions.Config, "config", "c", "", "absolute path to image registry configuration file")
```

Comment:
Can we keep the default config to be `~/.docker/config.json`? Will this create problem?

Reply:
currently if the file is not present/no content then it'll halt the execution by throwing an error. should we skip the check for error?

Reply:
@nyrahul i've updated the logic to make `~/.docker/config.json` as default and also made sure that if the file is not present/is empty then use empty username, password so that it'll still download public images.